.github/workflows/kaf.yaml

```
@@ -0,0 +1,54 @@
+# kaf.yaml - builds and pushes a Kaf container image.
+#
+# Requires repository variables:
+# - REGISTRY_SERVER - the server of the container registry service e.g. `quay.io` or `docker.io`
+# - REGISTRY_USERNAME - - your username on the service (or username of your robot account)
+# - REGISTRY_TOKEN - the access token that corresponds to `REGISTRY_USERNAME`
+#
+# If the required repository variables aren't set the workflow will be skipped.  This means the workflow won't fail
+# on the forks of developers who haven't configured the variables/secrets.
+
+name: Kaf Container 
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: 'Repository containing kaf'
+        default: 'birdayz/kaf'
+        required: true
+      registry-destination:
+        description: 'The release version, e.g. quay.io/<my org>/kaf'
+        default: 'quay.io/k_wall/kaf'
```

Err default in this repo?
```suggestion
        default: 'quay.io/kroxylicious/kaf'
```